build/distributions/CentOS/8/x86_64/bolt_pulp3_config.yaml

```diff
@@ -422,6 +420,10 @@ PowerTools:
     - name: elinks
     - name: lua-filesystem
     - name: lua-posix
+  modules:
+    - stream: javapackages-runtime:201801
+      rpms:
+        - javapackages-filesystem
```

Comment:
Investigated failure; looks like `javapackages-runtime` module is from AppStream, not PowerTools